### ISSUE TITLE
sqlite dependency

### ISSUE THREAD

**Issue by lastmjs** [aerogear/graphback#1412]
Hey, so I'm trying to deploy graphback into my server staging environment. For the subscriptions, I'm using a simple Ubuntu EC2 instance. I'm having a lot of trouble installing graphback because of the sqlite dependency, because it has to compile native modules and I'm having to install a lot of things to get that compilation to work.

As far as I know, I'm not using sqlite for anything, I'm attempting to connect to postgres, so why does graphback need it?

I think it would be a lot better to drop that dependency somehow if it isn't needed

**Comment by wtrocki** [comment]
Yep.. The best workaround is to edit your lock file and ignore sqllite. We will remove it
